sqlglot/dialects.py

```diff
@@ -461,25 +628,27 @@ def _unix_to_time(self, expression):
 Hive.functions = {
     "APPROX_COUNT_DISTINCT": exp.ApproxDistinct.from_arg_list,
     "COLLECT_LIST": exp.ArrayAgg.from_arg_list,
-    "DATE_ADD": lambda args: exp.TsOrDsAdd(
+    "DATE_ADD": lambda args: exp.MixedTypeAdd(
         this=list_get(args, 0),
         expression=list_get(args, 1),
         unit=exp.Literal.string("DAY"),
+        output_format=exp.Literal.string("YYYY-MM-DD"),
```

Comment:
so in this case, it should probably be Hive.format_time("YYYY-MM-DD") or something like that so that the output_format is in the generic form